CMakeLists.txt

```diff
@@ -30,19 +34,23 @@ function(packageProject)
     add_library(${PROJECT_NAMESPACE}${PROJECT_NAME} ALIAS ${PROJECT_NAME})
   endif()
 
-  if (DEFINED PROJECT_VERSION_HEADER)
+  if(DEFINED PROJECT_VERSION_HEADER)
     set(PROJECT_VERSION_INCLUDE_DIR ${PROJECT_BINARY_DIR}/PackageProjectInclude)
     string(TOUPPER ${PROJECT_NAME} UPPERCASE_PROJECT_NAME)
     configure_file(
       ${PACKAGE_PROJECT_ROOT_PATH}/version.h.in
       ${PROJECT_VERSION_INCLUDE_DIR}/${PROJECT_VERSION_HEADER}
       @ONLY
     )
+
+    # TODO: not clear to me why? CK
```

Comment:
@ClausKlein do you mean why we are differentiating between interface and library targets here?